pkg/build/controller/controller.go

```diff
@@ -208,7 +208,7 @@ func (bc *BuildController) resolveOutputDockerImageReference(build *buildapi.Bui
 		stream, err := bc.ImageStreamClient.GetImageStream(namespace, streamName)
 		if err != nil {
 			if errors.IsNotFound(err) {
-				return "", fmt.Errorf("the referenced output image stream %s/%s does not exist", namespace, streamName)
+				return "", fmt.Errorf("the referenced output image stream %s/%s does not exist.", namespace, streamName)
```

Comment:
we. are. same.

On Tue, Oct 27, 2015 at 3:55 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/build/controller/controller.go
> <https://github.com/openshift/origin/pull/5432#discussion_r43177064>:
>
> > @@ -208,7 +208,7 @@ func (bc *BuildController) resolveOutputDockerImageReference(build *buildapi.Bui
> >  		stream, err := bc.ImageStreamClient.GetImageStream(namespace, streamName)
> >  		if err != nil {
> >  			if errors.IsNotFound(err) {
> > -				return "", fmt.Errorf("the referenced output image stream %s/%s does not exist", namespace, streamName)
> > +				return "", fmt.Errorf("the referenced output image stream %s/%s does not exist.", namespace, streamName)
>
> @liggitt <https://github.com/liggitt> i have no idea why i read
> "jwforres" as liggit above. sorry. old age hits hard.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5432/files#r43177064>.
>